lib/screens_web/views/v2/audio/reconstructed_alert_single_screen_view.ex

```diff
@@ -220,4 +221,11 @@ defmodule ScreensWeb.V2.Audio.ReconstructedAlertSingleScreenView do
 
   defp render_cause(nil), do: nil
   defp render_cause(cause), do: " #{Alert.get_cause_string(cause)}"
+
+  defp issue_to_string(%{text: _} = issue) do
+    issueFreeText = struct(FreeTextLine, issue)
```

Comment:
I suspect Credo would be complaining about this Javascript-y variable name, if Actions wasn't taking the afternoon off.

Reply:
Oof. No idea what I was thinking here. Will address.